Azure/azure-container-networking
# Remove test coverage

**Reason for Change**
Removes components from go test coverage (Azure CLI, test utils/fakes, and generative code in CRD). 

Also updates a manifest yaml in CRD (via `make`)

**Issue Fixed**:

**Requirements**:
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)

**Notes**:
Tested the following make commands:
- make azure-npm-image
- make azure-cns-image
- make tools-images
- make test-all
  - this warning appears now (when using `go list ./...` for `-coverpkg`): `warning: no packages being tested depend on matches for pattern`

All images deployed to the cluster successfully, although I can't double check that CNS works right now, since even master is experiencing crashes right now (see error at bottom of log):
```
2021/09/14 21:40:50 [1] [Configuration] Config path:/etc/azure-cns/cns_config.json
2021/09/14 21:40:50 [1] GetAzureCloud querying url: http://169.254.169.254/metadata/instance/compute/azEnvironment?api-version=2018-10-01&format=text
2021/09/14 21:40:50 [1] [Azure CNS] Read config :{ChannelMode:CRD InitializeFromCNI:true ManagedSettings:{PrivateEndpoint: InfrastructureNetworkID: NodeID: NodeSyncIntervalInSeconds:30} MetricsBindAddress::9090 SyncHostNCTimeoutMs:500ns SyncHostNCVersionIntervalMs:1µs TLSCertificatePath: TLSEndpoint: TLSPort: TLSSubjectName: TelemetrySettings:{DisableAll:false DisableTrace:false DisableMetric:false DisableEvent:false TelemetryBatchSizeBytes:16384 TelemetryBatchIntervalInSecs:15 HeartBeatIntervalInMins:30 DisableMetadataRefreshThread:false RefreshIntervalInSecs:15 DebugMode:false SnapshotIntervalInMins:60} UseHTTPS:false WireserverIP:}
2021/09/14 21:40:50 [1] [Utils] Initializing HTTP client with connection timeout: 7, response header timeout: 7
2021/09/14 21:40:50 [1] AI Telemetry Handle created
2021/09/14 21:40:50 [1] [Telemetry] Request metadata from wireserver
2021/09/14 21:40:50 [1] Running on Linux version 5.4.0-1055-azure (buildd@lcy01-amd64-025) (gcc version 7.5.0 (Ubuntu 7.5.0-3ubuntu1~18.04)) #57~18.04.1-Ubuntu SMP Fri Jul 16 19:40:19 UTC 2021
2021/09/14 21:40:50 [1] [Azure CNS] Initialize HTTPRestService
2021/09/14 21:40:50 [1] HTTP listener will be started later after CNS state has been reconciled
2021/09/14 21:40:50 [1] [Azure CNS] restoreState
2021/09/14 21:40:50 [1] [Azure CNS]  Restored state, &{Location: NetworkType: OrchestratorType:KubernetesCRD NodeID: Initialized:false ContainerIDByOrchestratorContext:map[] ContainerStatus:map[] Networks:map[] TimeStamp:2021-09-14 21:40:22.785631824 +0000 UTC joinedNetworks:map[]}
2021/09/14 21:40:50 [1] [Azure CNS] Enter Restoring Network State
2021/09/14 21:40:50 [1] [Azure CNS] Store timestamp is 2021-09-14 21:40:22.778993614 +0000 UTC.
2021/09/14 21:40:50 [1] SetContext details called with: KubernetesCRD orchestrator nodeID 
2021/09/14 21:40:50 [1] Failed to query uptime, err:exec: "uptime": executable file not found in $PATH
2021/09/14 21:40:50 [1] [Utils] Initializing HTTP client with connection timeout: 5, response header timeout: 120
2021/09/14 21:40:50 [1] [Azure CNS]  Listening.
2021/09/14 21:40:50 [1] [Telemetry] Writing metadata to file failed: open /tmp/azuremetadata.json: no such file or directory
2021/09/14 21:40:50 [1] Initializing from CNI
2021/09/14 21:40:50 [1] Set GlobalPodInfoScheme 1
2021/09/14 21:40:50 [1] [Azure CNS] Starting request controller
2021/09/14 21:40:50 [1] [Azure CNS] setOrchestratorType
2021/09/14 21:40:50 [1] SetContext details called with: KubernetesCRD orchestrator nodeID 
2021/09/14 21:40:50 [1] [Azure CNS] saveState
2021/09/14 21:40:50 [1] [Azure CNS]  State saved successfully.
2021/09/14 21:40:50 [1] [azure-cns] Sent cns.Response {ReturnCode:Success Message:}.
2021/09/14 21:40:51 [1] NewCNSIPAMPoolMonitor: Create IPAM Pool Monitor
2021/09/14 21:40:51 [1] InitRequestController
2021/09/14 21:40:51 [1] CRD is not defined on cluster: the server could not find the requested resource (get nodenetworkconfigs.acn.azure.com aks-nodepool1-22301600-vmss000001)
```
